recipes/tinycthreadpool/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 2.8.12)
```

Comment:
Please, use the template from https://github.com/conan-io/conan-center-index/tree/master/docs/package_templates/cmake_package/all/test_package

CONAN_LIBS is no longer used and it's almost deprecated. Also, consider only Cmake targets for usage.